public/assets/icons/white/shield_outline.svg

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   version="1.1"
+   width="24"
+   height="24"
```

Comment:
They are svgs so its not the end of the world, but it would be better imo to have this the same 512 x 512 size as the others.